buildSrc/src/main/java/io/reactor/gradle/JavaConventions.java

```diff
@@ -64,7 +64,9 @@ private void applyJavaConvention(Project project) {
 					       "-Xlint:-serial",      // intentionally disabled
 					       "-Xlint:-options",     // intentionally disabled
 					       "-Xlint:-fallthrough", // intentionally disabled
-					       "-Xlint:-rawtypes"     // TODO enable and fix warnings
+					       "-Xlint:-rawtypes",     // TODO enable and fix warnings
+						   "-Xmaxwarns",
```

Comment:
suggestion: override the formatting and put on same line since 1000 applies to the flag?